modules/@apostrophecms/image/ui/apos/components/AposMediaUploader.vue

```diff
@@ -128,17 +130,15 @@ export default {
         });
 
         // Send up placeholders
-        for (const file of files) {
-          this.createPlaceholder(file);
-        }
+        [ ...files ].forEach(() => this.createPlaceholder(emptyDoc));
```

Comment:
Why creating a new array here?

Reply:
It's not an array (or it can be) but array like structure (that has no forEach). It's a safety.

Reply:
are you talking about `files` that is not an array?